src/OrchardCore.Modules/Orchard.ContentFields/Views/BooleanFieldSettings.Edit.cshtml

```diff
@@ -8,6 +18,21 @@
     </div>
 </fieldset>
 
+<div class="row">
+    <fieldset class="form-group col-sm-6">
+        <label asp-for="Editor">@T["What type of editor should be used?"]</label>
+        <select asp-for="Editor" class="form-control">
+            @foreach (var editorShape in editorShapes)
+            {
+                dynamic shape = Factory.Create(editorShape);
+                shape.Editor = Model.Editor;
+                @await DisplayAsync(shape)
+            }
+        </select>
+        <span class="hint">@T["The editor to use. eg: Number, Range"]</span>
```

Comment:
Number and Range don't really apply to a bool.
Also do we really want to list some samples which would already by in the drop down?